### DISCUSSION TITLE
Using both preset algorithm but customizing some components?

### DISCUSSION THREAD

**Discussion by AungS8430** [ant-design/ant-design#45188] [Q&A]
I want to use a darkAlgorithm but I don't like card design, I tried customizing it using design token, but it overrides the darkAlgorithm's style in other components that I didn't include in design token, how do I use both but override only in some component?
Here's my current ConfigProvider element.
```jsx
<ConfigProvider
    theme={{
        algorithm: theme.darkAlgorithm,
        components: {
            Card: {
                actionsBg: "#000000",
                extraColor: "#ffffff",
            },
        },
    }}
></ConfigProvider>```

**Comment by MadCcc** [comment]
Any problem with your code?
